doc/release/upcoming_changes/16841.changes.rst

```diff
@@ -0,0 +1,4 @@
+Output if ``dtype`` of ``linspace`` is int
+----------------------------------------------------
+- If the ``dtype`` of the ``linspace`` is int, round down it and casted it.
+- To use it, write ``np.linspace(..., dtype=np.int)``
```

Comment:
`dtype=int`.  I think `dtype=np.int` is discouraged now.

Reply:
This note doesn't really capture what actually changed - it's not `dtype=int` that changed, it's how negative arguments are handled that changed.